tsup.config.ts

```diff
@@ -38,7 +38,7 @@ const config: Options = {
 
   // Split the output into chunks. This is useful for tree-shaking.
   // https://tsup.egoist.dev/#code-splitting
-  splitting: true,
+  splitting: false,
```

Comment:
This is the actual configuration change which disables the code splitting 
 / "chunking" feature.